build.gradle

```diff
@@ -96,7 +96,20 @@ subprojects {
     afterEvaluate {
         if (tasks.findByName('docker')) {
             docker {
+                // Use cached image; only recreate if different
                 noCache false
+                labels(['org.opencontainers.image.source':'https://github.com/department-of-veterans-affairs/abd-vro'])
```

Comment:
`LABEL org.opencontainers.image.source=https://github.com/department-of-veterans-affairs/abd-vro` can now be removed from `Dockerfile`s like https://github.com/department-of-veterans-affairs/abd-vro/blob/8cdc31ffafc5bfa3959c8c80336a79cdcf3b7a31/app/src/docker/Dockerfile#L2